README.md

```diff
@@ -49,8 +49,19 @@ Contains the JSON schemas used to validate the sources in the integration tests.
 
 ## GTFS Realtime Data Structure
 
-This is still under discussion. [Please comment to share your thoughts](https://github.com/MobilityData/mobility-database-catalogs/issues/36). [You can view what is currently drafted in the working document](https://docs.google.com/document/d/1Mlz3AXHItInitsOEAKKi8hZV9d3t3gpFbyIZGm67ESU/edit#heading=h.i8ficnh0oxru).
-
+|Field Name|Type|Presence|Definition|                                                                     
+|-|-|-|-|
+| mdb_source_id |  Unique ID | System generated | Unique numerical identifier.      |
+| data_type     | Enum| Required| The data format that the source uses: `gtfs-rt-vp`, `gtfs-rt-tu`, or `gtfs-rt-sa`.                                                                                                |
+| provider     | Text | Required                   | A commonly used name for the transit provider included in the source.  
+| name        |  Text |Optional              | An optional description of the data source, e.g to specify if the data source is an aggregate of multiple providers
+| static_reference |  Array of Integers |Optional              | A list of the static sources that the real time source is associated with, represented by their MDB source IDs. |  
+|note|Text| Optional|A note to clarify complex use cases for consumers, for example when several static sources are represented in a realtime source.  |     
+|direct_download_url |URL|Optional     | URL that responds with an encoded [GTFS Realtime protocol buffer message](https://github.com/google/transit/tree/master/gtfs-realtime/spec/en#data-format).
```

Comment:
Your idea is great! I didn't think we would remove completely `realtime_vehicle_positions`, `realtime_trip_updates` and `realtime_alerts` from the schema - I thought a source could have only one of these. 

Two things if we adopt the `direct_download_url` as it is represented in the GTFS Schedule JSON Schema, i.e. as a `direct_download` URL field under a `urls` object. 

1. we need to represent the `entity_type` like mentioned above 
2. the `direct_download` should be represented as an object that would include the URL and everything related to the authentication, like:

```
{
    ...,
    "urls": {
        "direct_download": {
            "url": "some_url",
            "authentication": {
                ...
            }
        },
        "license": "some_license_url"
    },
    ...
}
```

If we decide to do this for `direct_download`, we would eventually need to do the same for the GTFS Schedule sources. Otherwise, I suggest keeping `realtime_vehicle_positions`, `realtime_trip_updates` and `realtime_alerts` as objects, which would include an URL and the authentication information.

Reply:
Since we want to include GTFS schedule URLs that have API authentication in the future, I think it makes sense to represent `direct_download` as an object rather than individual objects for the realtime entity types.